source/composeWith.js

```diff
@@ -0,0 +1,38 @@
+import pipeWith from './pipeWith';
+import reverse from './reverse';
+
+
+/**
+ * Performs right-to-left function composition using transforming function. The rightmost function may have
+ * any arity; the remaining functions must be unary.
+ *
+ * **Note:** The result of compose is not automatically curried.
+ *
+ * @func
+ * @memberOf R
+ * @category Function
+ * @sig ((y -> z), (x -> y), ..., (o -> p), ((a, b, ..., n) -> o)) -> ((a, b, ..., n) -> z)
```

Comment:
This signature looks like it might have been copied over from `compose`. I am somewhat at a loss for what the signature for this function _should_ be though...

Reply:
@Bradcomp can you help me with this?